server/src/lib/access_control/AccessController.spec.ts

```diff
@@ -362,6 +363,25 @@ describe("access_control/AccessController", function () {
         Assert(accessController.isAccessAllowed("home.example.com", "/dev/john", "john", ["dev"]));
         Assert(accessController.isAccessAllowed("home.example.com", "/dev/bob", "john", ["dev"]));
       });
+
+      it("should control whitelisting multiple networks to bypass authentication (real use case)", function () {
```

Comment:
I tried thinking about how to potentially write some integration tests though I was at a loss because we would need to know which source IP addresses the tests would be coming from, unless of course we decided to spoof the IP address by injecting them in headers with nginx or something else.

Reply:
Hi, 

Since in Configurator.ts you enable trust proxy, req.ip is already the left-most entry in the x-Forwarded-For header. See https://expressjs.com/en/api.html#req.ip

https://github.com/clems4ever/authelia/blob/a515ce83c758a38f9a32003d8d9182b533909473/server/src/lib/web_server/Configurator.ts#L39

Did this fail in some specific integration tests you were testing?